core/chain-configs/src/client_config.rs

```diff
@@ -410,6 +410,8 @@ pub struct ClientConfig {
     pub gc: GCConfig,
     /// Accounts that this client tracks.
     pub tracked_accounts: Vec<AccountId>,
+    /// Track shards that should be tracked by given validator.
+    pub shadow_tracked: Option<AccountId>,
```

Comment:
nit: How about tracked_shadow_validator? All the other relevant fields are prefixed tracked so I would stick with that and shadow alone doesn't describe well enough what this is about.